build/MUX-LocalizationHandoff.yml

```diff
@@ -16,26 +20,53 @@ jobs:
       nugetVer: true
       useDfs: false
 
-  - template: AzurePipelinesTemplates\MUX-PopulateBuildDateAndRevision-Steps.yml
+  - powershell: |
+      & "$env:Build_SourcesDirectory\build\Localization\PrepLocFilesForUpload.ps1" `
+        -DestinationFilePath '${{ variables.resourceStagingDirectory }}'
+    displayName: 'PrepLocFilesForUpload.ps1'
 
-  - task: CmdLine@1
-    displayName: 'Run Loc Workflow'
+  - task: TouchdownBuildTask@1
+    displayName: 'Call Touchdown Build Service'
     inputs:
-      filename: '$(Build.SourcesDirectory)\tools\VSTSRunLocWorkflow.cmd'
-    env:
-      MAPPED_LOC_KEY: $(WinUILab-TouchdownLoc-Key)
+      environment: 'PRODEXT'
+      teamId: '104'
+      authType: 'OAuth'
+      authId: 'e8416663-8a55-4cb6-9bba-12038b37a6ed'
+      authKey: $(WinUILab-TouchdownLoc-Key)
+      relativePathRoot: ${{ variables.resourceStagingDirectory }}
+      resourceFilePath: '**/Resources.resw'
+      outputDirectoryRoot: ${{ variables.resourceOutputDirectory }}
+      appendRelativeDir: true
+      cultureMappingType: 'None'
 
-  - powershell: |
-      $changes = & git ls-files -m
-      Write-Host "Edited files:"
-      Write-Host "$changes"
-      if ($changes -imatch "\.resw")
-      {
-        Write-Host "##vso[task.complete result=SucceededWithIssues;]"
-        Write-Host "##vso[task.logissue type=warning;]Localization updates needed"
-        foreach ($change in $changes)
-        {
-          Write-Host "##vso[task.logissue type=warning;]$change"
-        }
-      }
-    displayName: checkEditedFiles
\ No newline at end of file
+  - task: PublishBuildArtifacts@1
+    displayName: 'Publish Artifacts - Localized Resources'
+    condition: always()
+    inputs:
+      PathtoPublish: ${{ variables.resourceOutputDirectory }}
+      artifactName: 'LocalizationDrop'
+      parallel: true
+      parallelCount: 8
+
+  # Printing out list of files changed here currently does not reflect actual changes made.
```

Comment:
> # Printing out list of files changed here currently does not reflect actual changes made. [](start = 1, length = 90)

I don't really follow this. Can you update the comment, or remove this commented out code.